docker/runtime/backup-job.sh

```diff
@@ -16,7 +17,11 @@ execute_backup_job() {
 		BACKUP_ARGS="${BACKUP_ARGS} --exclude-repos=${BACKUP_ARG_EXCLUDE_REPOS}"
 	fi
 
-	./gogs backup "${BACKUP_ARGS}" || echo "Error: Backup job returned non-successful code." && exit 1
+	if [ -n "${BACKUP_EXTRA_ARGS}" ]; then
+		BACKUP_ARGS="${BACKUP_ARGS} ${BACKUP_EXTRA_ARGS}"
+	fi
+
+	./gogs backup ${BACKUP_ARGS} || echo "Error: Backup job returned non-successful code." && exit 1
```

Comment:
```suggestion
	./gogs backup "${BACKUP_ARGS}" || echo "Error: Backup job returned non-successful code." && exit 1
```

Is removing the double quota here on purpose (i.e. fix the bug you mentioned)? Because shell lint is complaining about it.

Reply:
Yes, thanks for reminding me and sorry for the late reply. 
I removed the double quotes here for bug-fix. It broken the shell lint rule, but we acturlly need to pass multi arguments in string to the backup command(rather than pass the hole string as one argument).

Reply:
typo: "the hole string" -> "the whole string"